commons/connector/health/aggregation/health_calories_measurements.avsc

```diff
@@ -0,0 +1,29 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKitCaloriesMeasurements",
+  "type": ["null", "record"],
+  "default": null,
+  "doc": "Caloires active, Calories basal",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit ."
+    },
+    {
+      "name": "calories.active",
```

Comment:
Please use camel case

Reply:
The same as above case, I have to specify it to be this way to get the data